### ISSUE TITLE
Provide option for pinning to taskbar

### ISSUE THREAD

**Issue by cinnamon-msft** [JaneaSystems/PowerToys-DevProjects#37]
I want this so bad

We can put it in the edit project view

**Comment by plante-msft** [comment]
Agreed this is important feature, but it has caveats (no easy way for us to do this without prompting for user consent). Consider as fast follow since users can create shortcut and pin shortcut to taskbar.

cc @SeraphimaZykova @donlaci
